### PR TITLE
Python improvements, example messaging, etc

### PR DESCRIPTION
**Pull request by transitive-bullshit**
- Add ability for python projects to specify examples which was previously breaking their saas clients
- Tweak messaging to OSS authors to be more succinct
- Some openapi fixes / cleanups
- Start prepping `wordcloud` for launch

WIP needs more testing

### CODE REVIEW

**Comment by transitive-bullshit** [comment]
Update: testing seems solid at this point. Will give another day or two to test more thoroughly.

Also really want to add some integration tests that deploy all the current examples to ensure we don't break anything obvious, but for now I'm just running these by hand.

File: examples/python/wordcloud/readme.md [hunk 1]
```diff
@@ -1,3 +1,49 @@
...
 
-TODO
+> Generate stylish wordclouds from any webpage.
+
+<a href="https://wordcloud.saasify.sh">
+  <img
+    src="https://raw.githubusercontent.com/saasify-sh/saasify/master/examples/python/wordcloud/examples/promo.png"
+    alt="Wordcloud Examples"
+  />
+</a>
```
Comment (chrisvxd) [inline examples/python/wordcloud/readme.md]: This thing isn't deployed yet

File: examples/python/wordcloud/readme.md [hunk 2]
```diff
@@ -1,3 +1,49 @@
...
+### The Onion
+
+```
+curl -X POST -o 'example.png' \
+  'https://api.saasify.sh/1/call/dev/wordcloud/stylecloud' \
+  -H 'content-type: application/json' \
+  -d '{ "url": "https://www.theonion.com/new-day-same-bullshit-whispers-dalai-lama-before-sl-1839720347" }'
+```
+
+<img src="https://raw.githubusercontent.com/saasify-sh/saasify/master/examples/python/wordcloud/examples/stylecloud.png" width="256" />
```
Comment (chrisvxd) [inline examples/python/wordcloud/readme.md]: Image is missing from PR, right?

File: packages/saasify-cli/lib/schemas/config.schema.json [hunk 3]
```diff
@@ -103,34 +101,13 @@
...
-          ]
-        },
         "inputContentType": {
           "type": "string",
           "title": "inputContentType"
         },
-        "outputContentType": {
-          "type": "string",
-          "title": "outputContentType"
-        },
```
Comment (chrisvxd) [inline packages/saasify-cli/lib/schemas/config.schema.json]: Do we want to issue a deprecation warning?

File: examples/python/wordcloud/readme.md (hunk 1 above)
Comment (transitive-bullshit) [reply examples/python/wordcloud/readme.md]: yep; there's an issue blocking the now deployment from working at the moment that I need to debug. something to do w/ the `Pipfile`

File: examples/python/wordcloud/readme.md (hunk 2 above)
Comment (transitive-bullshit) [reply examples/python/wordcloud/readme.md]: at the moment yep

File: packages/saasify-cli/lib/schemas/config.schema.json (hunk 3 above)
Comment (transitive-bullshit) [reply packages/saasify-cli/lib/schemas/config.schema.json]: nah; nobody's using this stuff aside from us. we'll def have to do this in the future though once we get more makers onboard.

**Pull request merged**